azavea/echo-locator
# Do not filter bounds layer on ECC toggle

## Overview

Always show all bounds on map; layer still updates to style routable vs non.


### Demo

![image](https://user-images.githubusercontent.com/960264/56830441-55e26800-6834-11e9-9386-50ffadf51bd5.png)

![image](https://user-images.githubusercontent.com/960264/56830461-6397ed80-6834-11e9-8956-be88372c3912.png)


## Testing Instructions

 * Sidebar results should filter appropriately on toggling show/hide non-ECC button
 * Bounds layer should always show all bounds on map
 * Bounds layer should update on button toggle to style/not style non-ECC polygons as routable
 * Clicking a routable (blue) polygon on the map should still go to details

Closes #165.
